stdlib/InteractiveUtils/src/InteractiveUtils.jl

```diff
@@ -351,4 +351,31 @@ function peakflops(n::Integer=2000; parallel::Bool=false)
     end
 end
 
+function report_bug(kind)
+    @info "Loading BugReporting package..."
+    BugReportingId = Base.PkgId(
+        Base.UUID((0xbcf9a6e7_4020_453c,0xb88e_690564246bb8)), "BugReporting")
+    # Check if the BugReporting package exists in the current environment
+    local BugReporting
+    if Base.locate_package(BugReportingId) === nothing
+        @info "Package `BugReporting` not found - attempting temporary installation"
+        # Create a temporary environment and add BugReporting
+        let Pkg = Base.require(Base.PkgId(
+            Base.UUID((0x44cfe95a_1eb2_52ea,0xb672_e2afdf69b78f)), "Pkg"))
+            return mktempdir() do tmp
+                prev_active = Base.ACTIVE_PROJECT[]
```

Comment:
If you want to handle the case where `@` is not in the load path you need something like
```
old_load_path = copy(LOAD_PATH)
push!(empty!(LOAD_PATH), tmp)
Pkg.add(...)
BugReporting = Base.require(...)
append!(empty!(LOAD_PATH), old_load_path)
```

Reply:
Doesn't it still need at least some project?
Without that
```
ERROR: no active project
Stacktrace:
 [1] pkgerror(::String) at /home/keno/julia-rebase/usr/share/julia/stdlib/v1.5/Pkg/src/Types.jl:52
 [2] find_project_file(::Nothing) at /home/keno/julia-rebase/usr/share/julia/stdlib/v1.5/Pkg/src/Types.jl:180
```
I'm happy to do whatever the recommended thing here, is, but I think I'll go ahead and merge as is, since it seems to work ok. Would appreciate a PR to do it the proper way :)